src/CodeCompliance/Domain/Checks/NotUnique/Method.php

```diff
@@ -71,6 +72,23 @@ public function getViolations(): array
         return $violations;
     }
 
+    /**
+     * @param \ReflectionClass $class
+     *
+     * @return bool
+     */
+    protected function isPlugin(ReflectionClass $class): bool
```

Comment:
Can we create a plugin filter which can be reused in the further checks?

Reply:
Added PluginFilter